Dockerfile

```diff
@@ -1,76 +0,0 @@
-# FROM java:8
-FROM alpine:3.7
-
-ENV SONAR_VERSION=6.1 \
-    SONARQUBE_HOME=/opt/sonarqube \
-    SONAR_RUNNER_VERSION=2.4 \
-    SONAR_RUNNER_HOME=/opt/sonar-runner \
-    # Database configuration
-    # Defaults to using H2
-    SONARQUBE_JDBC_USERNAME=sonar \
-    SONARQUBE_JDBC_PASSWORD=sonar \
-    SONARQUBE_JDBC_URL=
-
-ENV JAVA_HOME="/usr/lib/jvm/java-1.8-openjdk"
-
-# Http port
-EXPOSE 9000
-
-RUN apk update \
-    && apk upgrade \
-    && apk add --no-cache bash \
-    && apk add --no-cache --virtual=build-dependencies unzip \
-    && apk add --no-cache curl \
-    && apk add --no-cache openjdk8-jre \
```

Comment:
Maybe we can use an openjdk docker image instead of installing it.

Reply:
Also good idea for when we upgrade